modules/monitoring-setting-up-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
+
+:_content-type: CONCEPT
+[id="setting-up-alert-routing-for-user-defined-projects_{context}"]
+= Setting up alert routing for user-defined projects
+
+[role="_abstract"]
+As a cluster administrator, you can set up alert routing for user-defined projects. 
+After doing so, you can allow users to configure alert routing for their user-defined projects using either the default Alertmanager instance or the optional Alertmanager instance dedicated only to user workload monitoring, if enabled. 
```

Comment:
(nit) do we want to use "user workload monitoring" here? IIUC the official docs stick with "user-defined monitoring".